diff --git a/install.js b/install.js
index d97764edaa6e2ade24be5bcaa1c0e06064cb18b9..313b9dcfc848a992f011e99d2f09d516412d1ab3 100644
--- a/install.js
+++ b/install.js
@@ -230,7 +230,7 @@ function maybeOptimizePackage(binPath) {
   }
 }
 async function downloadDirectlyFromNPM(pkg, subpath, binPath) {
-  const url = `https://registry.npmjs.org/${pkg}/-/${pkg.replace("@esbuild/", "")}-${versionFromPackageJSON}.tgz`;
+  const url = `https://registry.npmmirror.com/${pkg}/-/${pkg.replace("@esbuild/", "")}-${versionFromPackageJSON}.tgz`;
   console.error(`[esbuild] Trying to download ${JSON.stringify(url)}`);
   try {
     fs2.writeFileSync(binPath, extractFileFromTarGzip(await fetch(url), subpath));
